attestation-agent/attestation-agent/src/bin/grpc-aa/main.rs

```diff
@@ -40,10 +40,23 @@ struct Cli {
     /// Initdata digest to be verified by AA. If initdata check failed, AA will failed to launch.
     /// The initdata should be base64 standard encoding.
     ///
+    /// Note that this is an alternative to `--initdata_toml`, and only one of them should be used.
+    ///
     /// Example:
     /// `--initdata_digest AAAAAAAAAAAA`
     #[arg(short, long)]
     initdata_digest: Option<String>,
+
+    /// Path to the Initdata TOML file to be verified by AA. If initdata check failed, AA will failed to launch.
+    /// The initdata should be base64 standard encoding.
+    ///
+    /// Note that this is an alternative to `--initdata_digest`, and only one of them should be used.
+    /// If this option is provided, the file will be read and its content will be used for verification.
+    ///
+    /// Example:
+    /// `--initdata_toml /path/to/initdata.toml`
+    #[arg(short = 't', long)]
+    initdata_toml: Option<String>,
```

Comment:
Apologies first of all. 
Perhaps I lack the related context or history of initdata, I just base on the Specification of `InitData`, and get its design of initdata which supports not only files with format `toml` but also `json`. To be honest, I am confused about it why we only support toml in code ? It came from the practice or to prevent some potential issues ?

Based on my understanding of InitData, what's about just using the `initdata` for the initdata plaintext and  `initdata-digest` for its digest ?

Reply:
Oh yes. The original [spec](https://github.com/confidential-containers/trustee/blob/main/kbs/docs/initdata.md#examples-for-different-encodings) of initdata declared supporting different encodings, i.e. initdata toml and initdata json, but we choose initdata toml in kata/coco.